### DISCUSSION TITLE
Plotting data availability problem

### DISCUSSION THREAD

**Discussion by EvanFadhilahR** [ROBelgium/MSNoise#330] [I think there is a bug]
I dont know what's the problem, but somehow the data plot cant show in picture
![Screenshot 2024-03-11 235011](https://github.com/ROBelgium/MSNoise/assets/163017878/6167e77f-f1e3-4ecc-b075-c109352377de)
![Screenshot 2024-03-11 235109](https://github.com/ROBelgium/MSNoise/assets/163017878/e028a272-c5f5-4810-84d3-51a012cbb7b1)

**Comment by ThomasLecocq** [comment]
Your loc ids are strange, what does the data availability table look like?

**Reply by EvanFadhilahR** [reply]
i tried to use sqlite, then how to check the data availability table sir?

**Reply by EvanFadhilahR** [reply]
maybe this is the data table that what you mean
![Uploading Screenshot 2024-03-12 140529.png…]()

**Reply by ThomasLecocq** [reply]
use the msnoise admin - note your channels are really non-standard, I can't guarantee msnoise will be happy/working correctly.
